lms/services/oauth_http.py

```diff
@@ -42,6 +45,41 @@ def request(self, method, url, headers=None, **kwargs):
 
         return self._http_service.request(method, url, headers=headers, **kwargs)
 
+    def access_token_request(self, token_url, redirect_uri, auth, authorization_code):
```

Comment:
```suggestion
    def get_access_token(self, token_url, redirect_uri, auth, authorization_code):
```

This is phrased a bit noun-y for my taste. This does not return a request but make one. I think `request_access_token` would work too.

Reply:
Yep, I went with `get_access_token()`